docs/overview.rst

```diff
@@ -87,8 +87,8 @@ the following operations are performed:
 1. find the backends where the database is stored
 2. find the latest version of a database (optional)
 3. calculate :ref:`database dependencies <database-dependencies>`
-4. download archive files from the selected backend
-5. unpack the archive files
+4. download (archive) files from the selected backend
```

Comment:
I wonder whether this is the right location for showing how (media) deps are obtained? Probably the documentation mentioned in 3. would be the better location as this would make things too complicated in this enumeration.  When a media file for a specific flavor is already in the cache it will be copied to the the requested flavor's media directory location, downloading and unpacking will not be necessary. At least as far as my understanding goes ...

Reply:
You are right, media files for a specific flavor are first looked for in caches of other versions of the same database with the same flavor:

https://github.com/audeering/audb/blob/0cc749747f5cfb75132876665b9c63bb54a2121f/audb/core/load.py#L693-L717

The same is also true for tables.

We added this feature at a later stage, and I'm pretty sure the documentation existed already, and we didn't bother updating it. I created https://github.com/audeering/audb/issues/439 to track this, and will not include it in this pull request.

Reply:
Great. So this can count as resolved here.

Reply:
Or maybe I got confused by the term `download`.   when files come via `_get_files_from_cache`, then the - but this can also be handled in the follow up. Leaving this in resolved state.